src/Monolog/Handler/SwiftMailerHandler.php

```diff
@@ -28,6 +28,8 @@ class SwiftMailerHandler extends MailHandler
     private $messageTemplate;
 
     /**
+     * @psalm-param callable(string, array): Swift_Message|Swift_Message $message
```

Comment:
This seems ambiguous especially in light of php8's union types.. Is this a callable which returns `Swift_Message OR Swift_Message`, or is it a `callable OR a Swift_Message` (that's the correct one of course, but it reads super weird IMO right now).

Reply:
I changed the order in all similar parts add first the `Swift_message` and then the `callable`

Reply:
Thanks yeah that's better :)